src/main/java/com/softserveinc/Task03.java

```diff
@@ -5,7 +5,53 @@
 public class Task03 {
     public static void main(String[] args) {
         Scanner scanner = new Scanner(System.in);
-        // TODO: Write your code here
 
+        System.out.println("Write how many ml of water the coffee machine has: ");
+        int water = (int) scanner.nextInt();
```

Comment:
You don't need to cast to int. It is redundant because the method nextInt() returns an integer.  
`
int water = scanner.nextInt();
`